client_app/src/pages/SignUpPage/SignUpPage.jsx

```diff
@@ -1,8 +1,197 @@
-import React from "react";
+import {
+  Container,
+  Grid,
+  Typography,
+  TextField,
+  Button
+} from "@material-ui/core";
+import useStyles from "./styles";
+import React, { forwardRef, useCallback } from "react";
+import { useHistory } from "react-router-dom";
+import IntlPhoneInput from "../../components/PhoneInput/PhoneInput";
+import "react-phone-number-input/style.css";
+import PhoneInput, { isValidPhoneNumber } from "react-phone-number-input";
+import flags from "react-phone-number-input/flags";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import { useTranslation } from "react-i18next";
+import { sendCreateAccountRequest } from "../../services/AuthenticationService";
+import useSessionStorage from "../../utils/Hooks/useSessionStorage";
+import { ROUTES } from "../../utils/enums";
+
+const IntlPhoneRef = forwardRef(IntlPhoneInput);
 
 export default function SignUpPage() {
+  const classes = useStyles();
+  const [ , setItem ] = useSessionStorage();
+  const { t } = useTranslation([ "Global", "SignUp" ]);
+  const history = useHistory();
+
+  const handleSubmit = async values => {
+    const [ result, error ] = await sendCreateAccountRequest(values);
+
+    if (error) {
+      // TODO Display error banner here RENT-57
+      return;
+    }
+    setItem("accountId", result.id);
+    history.push(ROUTES.VERIFY);
+  };
+
+  const formik = useFormik({
+    initialValues: {
+      firstName: "",
+      lastName: "",
+      email: "",
+      phoneNumber: "",
+    },
+    validationSchema: Yup.object({
+      firstName: Yup.string()
+        .max(50, "Must be 50 characters or less")
+        .required("Required"),
+      lastName: Yup.string()
+        .max(50, "Must be 20 characters or less")
+        .required("Required"),
+      email: Yup.string().email("Invalid email address").required("Required"),
+      phoneNumber: Yup.string().required("Required")
```

Comment:
To do utiliser les traductions pour les messages (ma pr va en ajouter plusieurs dans les fichiers)